app/components/VaultControls/index.js

```diff
@@ -999,8 +1002,29 @@ export default function VaultControls(props) {
                     defaultValue={withdrawTokens[0]}
                     value={selectedWithdrawToken}
                     options={withdrawTokens}
-                    onChange={(newValue) => {
+                    onChange={async (newValue) => {
                       setSelectedWithdrawToken(newValue);
+                      if (
+                        web3Account &&
+                        newValue.address !== currentVaultToken.address
+                      ) {
+                        const approvalStateRes = await fetch(
+                          `https://api.zapper.fi/v1/zap-out/yearn/approval-state?` +
+                            `api_key=96e0cc51-a62e-42ca-acee-910ea7d2a241&sellTokenAddress=` +
```

Comment:
Lets import the zapper key from .env `const { ZAPPER_APIKEY } = process.env;`

Reply:
done